code/datums/ert.dm

```diff
@@ -41,6 +41,13 @@
 	mission = "Leave no witnesses."
 	polldesc = "an elite Nanotrasen Strike Team"
 
+/datum/ert/marine
+	leader_role = /datum/antagonist/ert/marine
+	roles = list(/datum/antagonist/ert/marine/security, /datum/antagonist/ert/marine/engineer, /datum/antagonist/ert/marine/medic)
+	rename_team = "Marine Squad"
+	polldesc = "an elite Nanotrasen Strike Team"
```

Comment:
this probably shouldnt be the same message as the death squad

Reply:
it should be better now